### ISSUE TITLE
The content is locked by another user overlay is not shown correcly

### ISSUE THREAD

**Issue by grappler** [WordPress/gutenberg#3446]
## Issue Overview
When the content is locked by another user the overlay is not shown correctly when a plugin with a meta box like Yoast SEO is installed. When the plugin is not installed there is no overlay so there is no way of knowing that someone is editing the post.

## Steps to Reproduce (for bugs)
1. Open a post with one user account
2. Enable Yoast SEO
3. Open the same post with another user account
4. See the overlay in the Yoast SEO Meta Box

## Expected Behavior
The overlay is shown across the whole editor and not just in the Yoast SEO Meta Box

## Screenshots / Video
![image](https://user-images.githubusercontent.com/1785641/32702806-54a96cb6-c7ec-11e7-8b74-19caf8189105.png)

## Todos
- [ ] Tests
- [ ] Documentation

**Comment by StaggerLeee** [comment]
This is also one of the very strange things from "old" WordPress core.

How it is possible Editor can take over Post from Admin and kick Admin out of the Post ?
How it is possible the same Role can kick each other out of the Post ?

Maybe there is a reason I am not aware of. 
Heartbeat sholud "logout" an User from Post after some time of inactivity. If not User is editing and nobody other sholud meddle in affair.

**Comment by pento** [comment]
I can reproduce this issue in master, but it's actually fixed as a side effect of #3345, which changes the metabox handler to not use iframes, so I'm going to close this issue in favour of that PR.

Thanks for the bug report, @grappler!

**Comment by grappler** [comment]
Good to know that it will be fixed. Might have been best to close the issue once the PR is merged.

@pento does it make sense to create a new issue for Gutenberg to support the locked overlay. At the moment there is no way of knowing is someone else is editing the post.

**Comment by pento** [comment]
Hopefully, post locking will become redundant with the work on collaborative editing (#1930). So, no need for an issue at the moment, but we can revisit that later if it turns out we still need some sort of post locking interface.

**Comment by gziolo** [comment]
@pento collaborative editing is being developed as a plugin, and even if it would be a part of the Gutenberg it's still optional. I think we should open an issue to bring back post locking based on heartbeat anyway. /cc @mtias
